package.json

```diff
@@ -1,59 +1,59 @@
-  "main": "lib/main.js",
-  "scripts": {
-    "build": "ncc build src/action-ros-ci.ts -o dist",
-    "lint": "eslint src/action-ros-ci.ts",
-    "prettier": "prettier --write 'src/action-ros-ci.ts'",
-    "test": "tsc --noEmit && jest --coverage",
-    "pre-commit": "npm run test && npm run lint && npm run build"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ros-tooling/action-ros-ci.git"
-  },
-  "keywords": [
-    "actions",
-    "ros",
-    "ci"
-  ],
-  "author": "ROS 2 Tooling Working Group",
-  "license": "Apache-2.0",
-  "licenses": [
-    {
-      "type": "Apache-2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0"
-    }
-  ],
-  "bugs": {
-    "url": "https://github.com/ros-tooling/action-ros-ci/issues"
-  },
-  "dependencies": {
-    "@actions/core": "^1.2.5",
-    "@actions/exec": "^1.0.4",
-    "@actions/github": "^4.0.0",
-    "@actions/io": "^1.0.1"
-  },
-  "devDependencies": {
-    "@types/jest": "^26.0.13",
-    "@types/node": "^14.6.4",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
-    "@zeit/ncc": "^0.22.3",
-    "acorn": "^8.0.1",
-    "eslint": "^6.5.1",
-    "husky": "^4.3.0",
-    "jest": "^25.5.4",
-    "jest-circus": "^26.4.2",
-    "prettier": "2.1.1",
-    "ts-jest": "^25.5.1",
-    "typescript": "^3.9.7"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run pre-commit && git add dist/index.js"
-    }
-  }
+	"name": "action-ros-ci",
+	"version": "0.0.18",
+	"description": "GitHub Action compiling and testing a ROS 2 package",
+	"main": "lib/main.js",
+	"scripts": {
+		"build": "ncc build src/action-ros-ci.ts -o dist",
+		"fixup": "eslint . --fix --ignore-path .lintignore && prettier . --write --ignore-path .lintignore",
+		"lint": "eslint src/action-ros-ci.ts",
+		"test": "tsc --noEmit && jest --coverage"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/ros-tooling/action-ros-ci.git"
+	},
+	"keywords": [
+		"actions",
+		"ros",
+		"ci"
+	],
+	"author": "ROS 2 Tooling Working Group",
+	"license": "Apache-2.0",
+	"licenses": [
+		{
+			"type": "Apache-2.0",
+			"url": "http://www.apache.org/licenses/LICENSE-2.0"
+		}
+	],
+	"bugs": {
+		"url": "https://github.com/ros-tooling/action-ros-ci/issues"
+	},
+	"dependencies": {
+		"@actions/core": "^1.2.5",
+		"@actions/exec": "^1.0.4",
+		"@actions/github": "^4.0.0",
+		"@actions/io": "^1.0.1"
+	},
+	"devDependencies": {
+		"@types/jest": "^26.0.13",
+		"@types/node": "^14.6.4",
+		"@typescript-eslint/eslint-plugin": "^2.34.0",
+		"@typescript-eslint/parser": "^2.34.0",
+		"@zeit/ncc": "^0.22.3",
+		"acorn": "^8.0.1",
+		"eslint": "^6.5.1",
+		"husky": "^4.3.0",
+		"jest": "^25.5.4",
+		"jest-circus": "^26.4.2",
+		"lint-staged": "^10.3.0",
+		"prettier": "2.1.1",
+		"ts-jest": "^25.5.1",
+		"typescript": "^3.9.7"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
```

Comment:
Is `git add dist/index.js` intentionally removed?

Reply:
Yes it was. Previously adding index.js here would allow unstaged changes from the typescript file to make it into the generated javascript.

But some mucking around shows that I *do* need a `git add`.

Reply:
The original reason it was added was because we had cases where developers forgot to add the compiled js file and got confused when their ts changes didn't take effect.

Thanks for adding it back!